packages/inter-protocol/src/price/fluxAggregator.contract.js

```diff
@@ -77,10 +77,17 @@ export const start = async (zcf, privateArgs, baggage) => {
     marshaller,
   );
 
-  trace('got param governance');
-
   const governedApis = {
-    initOracle: fa.creatorFacet.initOracle,
+    /**
+     *
+     * @param {string} oracleId
+     */
+    addOracle: oracleId => fa.creatorFacet.initOracle(oracleId),
+    /**
+     *
+     * @param {string} oracleId
+     */
+    removeOracle: oracleId => fa.creatorFacet.deleteOracle(oracleId),
```

Comment:
The inconsistency between `addOracle` and `initOracle` (and likewise `removeOracle` and `deleteOracle`) seems like a little sand in the gears.

Tolerable, though, I suppose.